lib/checks/aria/aria-hidden-body.js

```diff
@@ -0,0 +1,6 @@
+var aria = 'aria-hidden';
+if (node && node.hasAttribute(aria)) {
```

Comment:
You need to ensure that the value is `true` - a value of `false`, while dumb, does not cause any problems